src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -688,9 +688,11 @@ ReactDOMComponent.Mixin = {
           }
           propValue = CSSPropertyOperations.createMarkupForStyles(propValue);
         }
-        var markup = null;
+        var markup = '';
         if (this._tag != null && isCustomComponent(this._tag, props)) {
-          markup = DOMPropertyOperations.createMarkupForCustomAttribute(propKey, propValue);
+          if (propKey != 'children' || typeof propValue === 'string' || typeof propValue === 'number') {
```

Comment:
Plain string comparison is not closure compiler compatible.

We should ignore it even if it's a string/number.

Otherwise this won't work:

```js
<custom-element>My Text Content</custom-element>
```